docs/streams/upgrade-guide.html

```diff
@@ -51,18 +50,6 @@ <h1>Upgrade Guide and API Changes</h1>
         <li> update your code and swap old code and jar file with new code and new jar file </li>
```

Comment:
Cannot comment above, but it say:
```
As an alternative, an offline upgrade is also possible. Upgrading from any versions as old as 0.10.0.x to {{fullDotVersion}} in offline mode require the following steps:
```
And `stop all old (e.g., 0.10.0.x) application instances`.

That's not correct any longer. Let's bump `0.10.0.x` to `0.11.0.

I am also wondering if we should whole paragraphs `Streams API changes in XXX` for at least all 0.10.0.x and 0.10.1.x releases. Maybe even more (ie, up-to-including 2.3.x, we we only support rolling-bounce upgrade from 2.4+ as oldest version to 4.0+ now), and add a sentence + link to eg `3.9` docs for these older versions?

\cc @lucasbru @bbejeck WDYT?